src/local_scheduler/local_scheduler.cc

```diff
@@ -491,8 +500,14 @@ void process_plasma_notification(event_loop *loop,
   free(notification);
 }
 
-void reconstruct_task_update_callback(Task *task, void *user_context) {
-  if (task == NULL) {
+void reconstruct_task_update_callback(Task *task,
+                                      void *user_context,
+                                      bool updated) {
+  /* The task ID should be in the task table for all tasks except the driver
+   * task. We should never request reconstruction for an object produced by the
+   * driver task. */
```

Comment:
That is, isn't the task ID in the task table for all tasks INCLUDING the driver task now?